### ISSUE TITLE
implementation status

### ISSUE THREAD

**Issue by yorkie** [yorkie/tensorflow-nodejs#3]
Providing TensorFlow functionality in a programming language can be broken down into broad categories:

- [x] Run a predefined graph
- [x] Graph construction
- [ ] Gradients (AKA automatic differentiation)
- [ ] Functions
- [x] Control Flow
- [ ] Neural Network library

**Comment by yorkie** [comment]
Actually I'm working on the 3rd part, namely Gradients in JavaScript, but it should be stoped now and get "predefined graph importing" is working firstly :)

**Comment by yorkie** [comment]
> Actually I'm working on the 3rd part, namely Gradients in JavaScript, but it should be stoped now and get "predefined graph importing" is working firstly :)

Predefined graph importing and exporting is working now, just revisit **Gradients** part :)
